app/views/bids/index.html

```diff
@@ -0,0 +1,29 @@
+<h1>Bids for "<%= @auction.title %>"</h1>
+
+<a href="/auctions/<%= @auction.id %>">« Back to auction</a>
+
+<table class="usa-table-borderless">
+  <thead>
+    <tr>
+      <th scope="col">Bidder</th>
+      <th scope="col">Amount</th>
+      <th scope="col">Date (Eastern Standard Time)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @auction.bids.sort_by {|b| b.created_at}.reverse.each_with_index do |bid, i| %>
+      <tr>
+        <td><%= bid.bidder_name %></td>
+        <% if i == 0 %>
+          <td><%= number_to_currency(bid.amount) %> &#42;</td>
```

Comment:
so for the first bid you want to append w ` &#42;` but none others? why is that?